docs/layer_configuration.md

```diff
@@ -0,0 +1,342 @@
+      * One of the standard operating system install paths
+      * It was added using one of the layer path environment variables (`VK_LAYER_PATH` or `VK_ADD_LAYER_PATH`).
+      * See the `Layer Discovery` section of the Vulkan Loader's [Layer Interface doc](https://github.com/KhronosGroup/Vulkan-Loader/blob/main/docs/LoaderLayerInterface.md).
+  2. The layer's library file is able to be loaded by the Vulkan Desktop Loader because it is in:
+      * A standard library path for the operating system
+      * The library path has been updated using an operating system-specific mechanism such as:
+          * Linux: adding the path to the layer's library .so with `LD_LIBRARY_PATH`
+          * MacOS: adding the path to the layer's library .dylib with `DYLD_LIBRARY_PATH` 
+  3. The layer's library file is compiled for the same target and bitdepth (32 vs 64) as the application
+
+#### Activating Specific SDK Layers
+
+To activate layers located in a particular SDK installation, or layers built locally from source, specify the layer JSON
+manifest file directory using either `VK_LAYER_PATH` or `VK_ADD_LAYER_PATH`.
+The difference between `VK_LAYER_PATH` and `VK_ADD_LAYER_PATH` is that `VK_LAYER_PATH` overrides the system layer paths
+so that no system layers are loaded by default unless their path is added to the environment variable.
+`VK_ADD_LAYER_PATH` on the otherhand, causes the loader to search the additional layer paths listed in the
+environment variable first, and then the standard system paths will be searched.
+
+#### Example Usage On Windows:
+
+For example, if a Vulkan SDK is installed in `C:\VulkanSDK\1.2.198.0`, execute the following in a Command Window:
+
+```
+C:\> set VK_LAYER_PATH=C:\VulkanSDK\1.2.198.0\Bin
+```
+
+#### Example Usage on Linux
+
+For Linux, if Vulkan SDK 1.2.198.0 was locally installed in `/sdk` and `VULKAN_SDK=/sdk/1.2.198.0/x86_64`:
+
+```
+$ export VK_LAYER_PATH=$VULKAN_SDK/lib/vulkan/layers
+$ export LD_LIBRARY_PATH=$VULKAN_SDK/lib:$VULKAN_SDK/lib/vulkan/layers
+```
+
+#### Example Usage on MacOS
+
+For macOS, if Vulkan SDK 1.2.198.0 was locally installed in `/sdk` and `VULKAN_SDK=/sdk/1.2.198/macOS`:
+
+```
+$ export VK_LAYER_PATH=$VULKAN_SDK/share/vulkan/explicit_layers.d
+$ export DYLD_LIBRARY_PATH=$VULKAN_SDK/lib
+```
+
+### Enabling and ordering Vulkan Layers
+
+Originally, the Vulkan Desktop Loader provided `VK_INSTANCE_LAYERS` to enable layers from the command-line.
+However, starting with the Vulkan Loader built against the 1.3.234 Vulkan headers, the `VK_LOADER_LAYERS_ENABLE` environment
+variable was added to allow for more easily enabling Vulkan layers.
+The newer Loaders will continue to accept the original `VK_INSTANCE_LAYERS` environment variable for some time, but it is
+considered deprecated.
+
+#### Vulkan 1.3.234 Loader and Newer (`VK_LOADER_LAYERS_ENABLE`)
+
+The easiest way to enable a layer with a more recent drop of the Vulkan Loader is using the `VK_LOADER_LAYERS_ENABLE
```

Comment:
right highlighter for VK_LOADER_LAYERS_ENABLE is missing

Reply:
fixed